src/html.rs

```diff
@@ -574,6 +573,15 @@ impl<'s, 'f> Writer<'s, 'f> {
                     Container::Math { display } => {
                         out.write_str(if *display { r#">\["# } else { r#">\("# })?;
                     }
+                    Container::TaskListItem { checked } => {
+                        out.write_str(">\n")?;
+                        self.indent(&mut out)?;
```

Comment:
Need to also handle when using minified, there should be no whitespace in that case. Should be able to use `self.block(&mut out, 0)` to conditionally write a newline. It also accounts for the indentation.